route/route.go

```diff
@@ -678,8 +681,10 @@ func (r *Router) batchedEventToEvent(req *http.Request, bev batchedEvent, apiKey
 	eventTime := bev.getEventTime()
 	// TODO move the following 3 lines outside of this loop; they could be done
 	// once for the entire batch instead of in every event.
-	vars := mux.Vars(req)
```

Comment:
Is this the time where we actually implement this TODO given that we're messing with this code?

Reply:
Moved apihost and dataset lookup outside of the loop in https://github.com/honeycombio/refinery/pull/1199/commits/458d27f3c97ddbfa44c59771a4ea17c875bfafd0